### ISSUE TITLE
Suggestion for new markup of crossed out math

### ISSUE THREAD

**Issue by MartheGjelstad** [nlbdev/mathml-guidelines#71]
The markup of "crossed out math" in our guidelines is not valid in Webarch. Therefore, Textalk has looked at an alternative solution for the markup, see the attached image. 

I was told the following: 
- "aria-describedby" should be used for shorter descriptions, while "aria-details" should be used for longer. 
- The aria-details value should match the id on the details tag. 
- In our current guidelines, the image is marked up as an inline element but have display set to block, which is somewhat contradictory. 

<img width="825" height="171" alt="Image" src="https://github.com/user-attachments/assets/51d1c49d-372f-4e52-a85b-3836a8354704" />
